lms/templates/courseware/info.html

```diff
@@ -54,7 +54,7 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
 <section class="container">
   <div class="home">
     <div class="page-header-main">
-        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.id.org, course_name=course.id.course) | h}</h1>
+        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default) | h}</h1>
         <h2 class="page-subtitle">${course.display_name | h}</h2>
```

Comment:
Ok.  You will need to leave the `| h` on this line, but you should switch to the safe by default `_()` and follow the rest of my instructions related to that change.